src/resources/PowerShellWorkerStrings.resx

```diff
@@ -352,7 +352,28 @@
   <data name="DependencySnapshotDoesNotContainAcceptableModuleVersions" xml:space="preserve">
     <value>Dependency snapshot '{0}' does not contain acceptable module versions.</value>
   </data>
+  <data name="FoundExternalDurableSdkInSession" xml:space="preserve">
+    <value>Found external Durable Functions SDK in session: Name='{0}', Version='{1}', Path='{2}'.</value>
+  </data>
+  <data name="UnableToInitializeOrchestrator" xml:space="preserve">
+    <value>Unable to initialize orchestrator function due to presence of other bindings. Total number of bindings found is '{0}'. Orchestrator Functions should never use any input or output bindings other than the orchestration trigger itself. See: aka.ms/df-bindings</value>
+  </data>
   <data name="WorkerInitCompleted" xml:space="preserve">
     <value>Worker init request completed in {0} ms.</value>
   </data>
+  <data name="UnexpectedResultCount" xml:space="preserve">
+    <value>Operation '{0}' expected '{1}' result(s) but received '{2}'.</value>
+  </data>
+  <data name="ExternalSDKWasNotLoaded" xml:space="preserve">
+    <value>The external Durable Functions SDK is enabled but it cannot be used because it was not loaded onto the current PowerShell session. Please add `Import-Module -Name {0} -ErrorAction Stop` to your profile.ps1 so it may be used. Defaulting to built-in SDK.</value>
+  </data>
+  <data name="MultipleExternalSDKsInSession" xml:space="preserve">
+    <value>Get-Module returned '{0}' instances of '{1}' in the PowerShell session, but only 1 or 0 are expected. This may create runtime errors. Please ensure your script only imports a single version of '{0}'. The modules currently in session are:\n '{2}'.</value>
```

Comment:
Should we include the `profile.ps1` file name in this line `Please ensure your script only imports a single version of '{0}'` ?

Reply:
Sure, I added an explicit reference to `profile.ps1` in my latest commit: https://github.com/Azure/azure-functions-powershell-worker/pull/839/commits/8e0e103d4ffa7abd758eab6581f9e96809049533

Does that look good?